lib/baz/collection_events/collection_event.ex

```diff
@@ -47,13 +48,15 @@ defmodule Baz.CollectionEvents.CollectionEvent do
       :event_type,
       :venue,
       :slug,
-      :token_id
+      :token_id,
+      :id
     ])
     |> validate_required([
       :event_timestamp,
       :event_type,
       :venue,
-      :slug
+      :slug,
+      :id
```

Comment:
Can we call this `venue_id`?

Reply:
It's event id.    But in the context of event, `id` is good enough.